app/controllers/dashboards_controller.rb

```diff
@@ -2,14 +2,14 @@ class DashboardsController < ApplicationController
   before_action :authenticate_user!, :set_user
 
   def show
-    @queries = @user.queries.page(params[:page]).per(10)
+    @queries = @user.queries #.page(params[:page]).per(10)
   end
 
   def search
     @queries = Query.search(
       search_params[:query],
-      where: { user_id: current_user.id },
-      page: params[:page], per_page: 10
+      where: { user_id: current_user.id } #,
+      #page: params[:page], per_page: 10
```

Comment:
Incorrect indentation detected (column 6 instead of 4).<br>Missing space after #.